builds/azure-pipelines/template-steps-publish.yml

```diff
@@ -6,6 +6,7 @@ steps:
   displayName: 'Install .NET Core SDK 5.0.x for CodeSign'
   inputs:
     version: '5.0.x'
+  condition: succeededOrFailed()
```

Comment:
Will this still try to codesign even if there was a compilation error?

Reply:
Yes, but given that it's already failing then it doesn't really matter if this step also fails IMO. Trying to determine exactly what failed and only do these steps on certain types of failures isn't really worth it IMO.